docs/en/install.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1. Setup Kubernetes cluster
 
+> Note: This should be prepared by platform builder
```

Comment:
I don't think it is necessary to add this line..
Just tell what to do here and don't say who is responsible for this.

Reply:
yes